.github/workflows/main.yml

```diff
@@ -0,0 +1,44 @@
+name: ci
+on: [push, pull_request]
+
+jobs:
+  build_and_deploy:
+    env:
+        BASE_VERSION: 5.0.0.2012
+    runs-on: windows-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Download Nuget.exe
+      uses: NuGet/setup-nuget@v1.0.5
+      with:
+        nuget-version: 5.4.0
+    - name: setup-msbuild
+      uses: microsoft/setup-msbuild@v1
+    - name: Restore & Build
+      run: |
+        cd Xamarin.Forms.Platform.GTK
+        Nuget.exe restore Xamarin.Forms.Platform.GTK.csproj
```

Comment:
nit: if you're putting msbuild as lower-case, I prefer you do same for `nuget.exe`